isotovideo

```diff
@@ -116,6 +127,13 @@ $| = 1;
 $bmwqemu::scriptdir = $installprefix;
 bmwqemu::init();
 
+for my $arg (@ARGV) {
+    if ($arg =~ /^([[:alnum:]_\[\]\.]+)=(.+)/) {
+        $bmwqemu::vars{uc $1} = $2;
+        diag("Setting forced test parameter $1 -> $2");
```

Comment:
I would expect the uc version in debug output

Reply:
done, updated.

Looks like this now:

```
isotovideo -d _exit_after_schedule=1 casedir=/var/lib/openqa/share/tests/sle prjdir=/var/lib/openqa/share productdir=/var/lib/openqa/share/tests/sle/products/sle' 
[2018-04-24T22:05:31.0923 CEST] [debug] Current version is 77d699efe03c42e466d3ebb6125e45f7cdcd2345 [interface v9]
[2018-04-24T22:05:31.0926 CEST] [debug] Setting forced test parameter _EXIT_AFTER_SCHEDULE -> 1
[2018-04-24T22:05:31.0926 CEST] [debug] Setting forced test parameter CASEDIR -> /var/lib/openqa/share/tests/sle
[2018-04-24T22:05:31.0926 CEST] [debug] Setting forced test parameter PRJDIR -> /var/lib/openqa/share
[2018-04-24T22:05:31.0926 CEST] [debug] Setting forced test parameter PRODUCTDIR -> /var/lib/openqa/share/tests/sle/products/sle
```